pkg/scheduler/framework/plugins/dynamicresources/dynamicresources.go

```diff
@@ -827,6 +888,24 @@ func (pl *DynamicResources) bindClaim(ctx context.Context, state *stateData, ind
 				claim = updatedClaim
 			}
 			claim.Status.Allocation = allocation
+
+			// if the claim has devices with binding conditions, we need to store them in the claim status
```

Comment:
as discussed above, I am not clear on why? Shouldn't we be able to use the values from the allocation result? they won't ever change